e2e/libs/backupstore/base.py

```
@@ -111,10 +115,18 @@ def update_backupstore(self,
+        for i in range(self.retry_count):
+            logging(f"Updating backupstore to {url}${credential_secret} ... ({i})")
+            try:
+                backup_target.backupTargetUpdate(name=self.DEFAULT_BACKUPTARGET,
+                                                 backupTargetURL=url,
+                                                 credentialSecret=credential_secret,
+                                                 pollInterval=str(poll_interval))
+                return
+            except Exception as e:
+                logging(f"Failed to update backupstore to {url}${credential_secret}: {e}")
+            time.sleep(self.retry_interval)
+        assert False, f"Failed to update backupstore to {url}${credential_secret}"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)